Resources/Prototypes/DeltaV/Entities/Objects/Weapons/Bombs/gingerbreadc4.yml

```diff
@@ -0,0 +1,63 @@
+﻿- type: entity
+  name: ginger-bomb man
+  description: Used to put holes in walls for naughty children.
+  parent: BaseItem
```

Comment:
bruh why not parent c4

Reply:
true, lemme fix that.

Reply:
and remove any duplicated fields

Reply:
> and remove any duplicated fields

what is that?

Reply:
things like Sticky, triggering on signal, etc
anything the c4 prototype has already you dont need, only keep the differences

Reply:
Oh right, I can do that.